packages/material-ui/src/Accordion/Accordion.d.ts

```diff
@@ -72,8 +60,6 @@ export interface AccordionProps extends StandardProps<PaperProps, 'onChange'> {
   TransitionProps?: TransitionProps;
 }
 
-export type AccordionClassKey = keyof NonNullable<AccordionProps['classes']>;
```

Comment:
I think these are used in the `overridse.d.ts`, but that is another story, we should revisit it, as not all classes support overrides, for example `disabled` support class overrides, but not style override

Reply:
The type still exists and did not change in this PR.

Reply:
Oh, I see, the type was moved to the other file.

Reply:
I've missed that too, sorry :)

Reply:
> I've missed that too, sorry :)

Nah, this is good feedback. I thought about mentioning `AccordionClasses` but not the `AccordionClassKey`.